en/docs/develop/restapis/email-template.yml

```diff
@@ -0,0 +1,588 @@
+        Adds a new email template type to the system. An email template type can have any number of email templates.
+
+        * Attribute _**displayName**_ of the template type should be unique.
+
+        <b>Permission required:</b>
+        * /permission/admin/manage/identity/emailmgt/create
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/EmailTemplateType'
+        description: Email template type to be added.
+      responses:
+        '201':
+          description: Item Created
+          headers:
+            self:
+              description: Location of the newly created identity provider
+              schema:
+                type: string
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/EmailTemplateTypeWithoutTemplates'
+        '400':
+          $ref: '#/components/responses/InvalidInput'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '500':
+          $ref: '#/components/responses/ServerError'
+  '/email/template-types/{template-type-id}':
+    get:
+      tags:
+        - Email Template Types
+      summary: Retrieve the email template type corresponds to the template type id.
+      operationId: getEmailTemplateType
+      description: |
+        Retrieve the email template type in the system identified by the template-type-id.
+
+        <b>Permission required:</b>
+        * /permission/admin/manage/identity/emailmgt/view
+      parameters:
+        - $ref: '#/components/parameters/templateTypeIdPathParam'
+        - $ref: '#/components/parameters/limitQueryParam'
+        - $ref: '#/components/parameters/offsetQueryParam'
+        - $ref: '#/components/parameters/sortOrderQueryParam'
+        - $ref: '#/components/parameters/sortByQueryParam'
+      responses:
+        '200':
+          description: Search results matching the given criteria.
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/EmailTemplateTypeWithID'
+        '400':
+          $ref: '#/components/responses/InvalidInput'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    post:
+      tags:
+        - Email Templates
+      summary: Add a new email template to an existing email template type.
+      operationId: addEmailTemplate
+      description: |
+        Adds a new email template an existing email template type in the system.
+        The locale of the new email template should not already exists in the
+        respective email template type.
```

Comment:
This sentence is not very clear..does it mean that the locale is not specified in the email template type and should be included in the email template?